tiledb/sm/c_api/tiledb.h

```diff
@@ -1077,7 +1077,7 @@ TILEDB_EXPORT int32_t tiledb_array_schema_set_domain(
  * @param array_schema The array schema.
  * @param capacity The capacity of a sparse data tile. Note that
  * sparse data tiles exist in sparse fragments, which can be created
- * in both sparse and dense arrays. For more details,
+ * in sparse arrays only. For more details,
```

Comment:
How about you remove the `Note…` part from here and instead update the function's description some lines earlier to `Sets the tile capacity. Applies to sparse arrays only.`?

Reply:
I find no reason to remove information, but let's see what @ihnorton has to say.